Asterix/main_THD.py

```diff
@@ -16,30 +16,19 @@ class THD2(Testbed):
         A read-in .ini parameter file.
     """
 
-    def __init__(self, parameter_file_path, new_model_config={}, new_dm_config={}, new_corona_config={}):
+    def __init__(self, config, model_local_dir):
         """
         Parameters
         ----------
-        parameter_file_path : string
-            Absolute path to an .ini parameter file.
-        new_model_config : dict, optional
-            Can be used to directly change a parameter in the MODELconfig section of the input parameter file.
-        new_dm_config : dict, optional
-            Can be used to directly change a parameter in the DMconfig section of the input parameter file.
-        new_corona_config : dict, optional
-            Can be used to directly change a parameter in the Coronaconfig section of the input parameter file.
+        config: dict
+            Parameter dictionary. Must at least contains parameter for Entrance Pupil, Coronagraphs and DM(s).
```

Comment:
Missing white space before colon; I would replace docstring with (fixing typos): "Testbed parameter dictionary. Must at least contain parameters for entrance pupil, coronagraph and DM(s)."

Reply:
About the semi collumn, apparently google style formatting is without semi collumn
https://google.github.io/styleguide/pyguide.html#doc-function-args

Reply:
updated the comments